todo_app/data/view_model.py

```diff
@@ -0,0 +1,32 @@
+from todo_app.data.item import Item
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        output = []
+        for item in self._items:
+            if item.status == "To Do":
+                output.append(item)
+        return output
+    
+    @property
+    def doing_items(self):
+        output = []
+        for item in self._items:
+            if item.status == "Doing":
+                output.append(item)
+        return output
```

Comment:
Not a recommendation, but I do often think it's interesting to compare this with the list-comprehension approach, which is (arguably) harder to read but more concise:
```python
    output = [item for item in self._items if item.status == "Doing"]
    return output
```